### PR TITLE
Replace CSS SVGs with files

### PR DESCRIPTION
**Pull request by kuasha420**
## Summary

Addresses issue:

- #7542

## Relevant technical choices

* The external icon present on Figma is different (crispier) than the one present on the CSS, hence the most of the VRT updates. 
* The implementation is not fully covered in the IB, and I used a wrapper to center the icon with the text using flexbox. If there's any better way to do it, I'd appreciate the suggestion and make the change. 

## PR Author Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 5.2 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [ ] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [ ] Run the code.
- [ ] Ensure the acceptance criteria are satisfied.
- [ ] Reassess the implementation with the IB.
- [ ] Ensure no unrelated changes are included.
- [ ] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [ ] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

File: assets/sass/components/global/_googlesitekit-icon-wrapper.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/**
+ * HelpMenuLink styles.
```
Comment (10upsimon) [inline assets/sass/components/global/_googlesitekit-icon-wrapper.scss:2]: I think this needs to be updated to be reflective of the `IconWrapper` name as opposed to `HelpMenuLink`.

File: assets/sass/components/global/_googlesitekit-cta-link.scss [hunk 2]
```diff
@@ -19,6 +19,7 @@
 .googlesitekit-cta-link {
 	color: $c-link;
```
Comment (10upsimon) [inline assets/sass/components/global/_googlesitekit-cta-link.scss:20]:
```suggestion
.googlesitekit-cta-link {
	align-items: center;
	color: $c-link;
```

@kuasha420 adding `align-items: center;` to the wrapper element seems to result in better vertical alignment. Attached are two screenshots showing the current alignment vs what it looks like with `align-items: center;` added:

**Without (current)**
Site Kit by Google Dashboard ‹ sitekit 10uplabs com — WordPress 2023-11-13 at 10 29 40 AM

**With `align-items: center;`**
Site Kit by Google Dashboard ‹ sitekit 10uplabs com — WordPress 2023-11-13 at 10 28 51 AM

**Review by 10upsimon — CHANGES_REQUESTED** [review]
@kuasha420 I some observations, particularly around better vertical alignment of the icon. 

I note that the icon _appears_ a bit smaller in the updated version, but I am not sure if that is simply because it is crisper and "less fuzzy" than the prior, but it appears this is lifted from Figma, so all good there I'd imagine.

**Review by tofumatt — CHANGES_REQUESTED** [review]
It looks like `hideExternalIndicator` was added to nearly every/every `external` link.

Should the behaviour of the `external` prop be changed? Having this many overrides seems like we're doing something wrong and it isn't useful anymore, so maybe we should change how it behaves in the `Link` logic instead of needing to add the `hideExternalIndicator` prop for every instance.

Otherwise looks good and seems to work well, but just curious about that prop 🤔

File: assets/js/components/KeyMetrics/GetHelpLink.js [hunk 3]
```diff
@@ -38,7 +38,7 @@ export default function GetHelpLink( { linkURL } ) {
 		__( 'Trouble getting access? <HelpLink />', 'google-site-kit' ),
 		{
 			HelpLink: (
-				<Link href={ linkURL } external>
+				<Link href={ linkURL } external hideExternalIndicator>
```
[the remark this answers is not in the document]
Comment (kuasha420) [reply assets/js/components/KeyMetrics/GetHelpLink.js:41]: Hi, all instances in this PR where I used the `hideExternalIndicator`, their external icons were being hidden using CSS, so I just replicated the old behavior with the prop. 

As for existence of the prop and usage, it predates this refactoring PR. In most cases, we want the "open in new tab" behavior of the `external` prop but not the icon. 

I think we can revisit the name and existence of the `external` and `hideExternalIndicator` props (maybe a `openInNewTab` prop instead) in a future issue and merge this as is, as any delay in merging this PR causes a bunch of conflicts with the VRT suite and eats more time.

What do you think?

**Review by tofumatt — APPROVED** [review]

**Pull request merged**